src/GeographicalFeatures/CriticalFacilities/index.js

```diff
@@ -377,6 +384,30 @@ class FeatureList extends Component {
     this.setState({ showFilters: false });
   };
 
+  /**
+   * @function
+   * @name handleView
+   * @description Handle on view item details action for list item
+   * @param {object} item item to be viewed
+   * @version 0.1.0
+   * @since 0.1.0
+   */
+  handleView = (item) => {
+    selectFeature(item);
+    this.setState({ showDetails: true });
+  };
+
+  /**
+   * @function
+   * @name closeDetails
+   * @description close feature details drawer
+   * @version 0.1.0
+   * @since 0.1.0
+   */
+  closeDetails = () => {
```

Comment:
Hello @Mloweedgar, Thanks for this PR.

Can you prefix these handler function with `handle` like what you  did with the handleView?